pkg/virt-controller/watch/drain/disruptionbudget/disruptionbudget.go

```diff
@@ -538,6 +538,9 @@ func (c *DisruptionBudgetController) sync(key string, vmiExists bool, vmi *virtv
 			// vmi isn't set to prevent eviction, so delete the pdb
 			log.Log.Object(vmi).Infof("deleting pdb %s/%s due to not using evictionStrategy: LiveMigration|External", pdb.Namespace, pdb.Name)
 			return c.deletePDB(key, pdb, vmi)
+		} else if vmi.IsFinal() && vmi.DeletionTimestamp == nil {
```

Comment:
I know @enp0s3 suggested to add the `vmi.DeletionTimestamp == nil` condition, but I would actually argue that this case is already taken care of by line 533 so actually it does not make much sense to replicate it here as it's going to always be true (otherwise you'd get inside the if at line 533).

Reply:
my thoughts exactly